docs/REST API/rest-api.md

```diff
@@ -6376,32 +6377,72 @@ None.
 }
 ```
 
-### Get Apple Business Manager (ABM)
+### List Apple Business Manager (ABM) connections
 
 _Available in Fleet Premium_
 
-`GET /api/v1/fleet/abm`
+`GET /api/v1/fleet/apple_business_manager`
 
 #### Parameters
 
 None.
 
 #### Example
 
-`GET /api/v1/fleet/abm`
+`GET /api/v1/fleet/apple_business_manager`
 
 ##### Default response
 
 `Status: 200`
 
 ```json
-{
-  "apple_id": "apple@example.com",
-  "org_name": "Fleet Device Management",
-  "mdm_server_url": "https://example.com/mdm/apple/mdm",
-  "renew_date": "2023-11-29T00:00:00Z",
-  "default_team": ""
-}
+[
+  {
+    "apple_id": "apple@example.com",
+    "org_name": "Fleet Device Management Inc.",
+    "mdm_server_url": "https://example.com/mdm/apple/mdm",
+    "renew_date": "2023-11-29T00:00:00Z",
+    "terms_expired": false,
+    "macos_team": "💻 Workstations",
+    "ios_team": "📱🏢 Company-owned iPhones",
+    "ipados_team": "🔳🏢 Company-owned iPads"
+  }
+]
+```
+
+### List Volume Purchasing Program (VPP) connections
+
+_Available in Fleet Premium_
+
+`GET /api/v1/fleet/volume_purchasing_program`
```

Comment:
We can break the existing `GET /vpp` endpoint. It's tagged as experimental: https://github.com/fleetdm/fleet/pull/19291/files#diff-7246bc304b15c8865ed8eaa205e9c244d0a0314e4bae60cf553dc06147c38b64R6395